src/bioutils/seqfetcher.py

```diff
@@ -139,11 +139,26 @@ def _fetch_seq_ensembl(ac, start_i=None, end_i=None):
         True
     """
 
+    # Ensembl API only takes transcript IDs (without version) and returns the latest one
+    # So we need to strip the transcript version, then check if what was returned was the one we requested
+    version = 0
+    m = re.match(r"^(ENST\d+)\.(\d)$", ac)
+    if m:
+        ac, version = m.groups()
+        version = int(version)
+
     url_fmt = "http://rest.ensembl.org/sequence/id/{ac}"
```

Comment:
Preferred (by me): `url = f"http://rest.ensembl.org/sequence/id/{ac}"`

Reply:
So do I - but that's a 3.8 feature - and setup has ```python_requires = >= 3.6``` so that will break 3.6

Reply:
Ah, looked and it's a 3.6 feature so that's ok